src/core/account_manager.mjs

```diff
@@ -115,36 +118,54 @@ export class AccountManager {
     return accountInfo
   }
 
-  /**
-   * Prepares the accounts with updated keys so that they do not contain the default genesis keys
-   * @param namespace the namespace to run the update of account keys for
-   * @returns {Promise<void>}
-   */
-  async prepareAccounts (namespace) {
-    const serviceMap = await this.getNodeServiceMap(namespace)
+  batchAccounts () {
+    const batchSize = constants.ACCOUNT_CREATE_BATCH_SIZE
+    const batchSets = []
 
-    const nodeClient = await this.getNodeClient(
-      namespace, serviceMap, constants.OPERATOR_ID, constants.OPERATOR_KEY)
-
-    await this.updateSpecialAccountsKeys(namespace, nodeClient, constants.SYSTEM_ACCOUNTS)
-    // update the treasury account last
-    await this.updateSpecialAccountsKeys(namespace, nodeClient, constants.TREASURY_ACCOUNTS)
+    let batchCounter = 0
+    let currentBatch = []
+    for (const [start, end] of constants.SYSTEM_ACCOUNTS) {
+      for (let i = start; i <= end; i++) {
+        if (batchCounter >= batchSize) {
+          batchSets.push(currentBatch)
+          currentBatch = []
+          batchCounter = 0
+        }
+        batchCounter++
+        currentBatch.push(i)
+      }
+    }
+    batchSets.push(currentBatch)
+    batchSets.push([constants.TREASURY_ACCOUNT])
 
-    nodeClient.close()
-    await this.stopPortForwards()
+    return batchSets
   }
 
   /**
    * stops and closes all of the port forwards that are running
    * @returns {Promise<void>}
    */
   async stopPortForwards () {
+    global.accountManagerPortForwardClosedCount = 0 // global variable to be used for WebSocketServer callback
```

Comment:
We should avoid global namespace, instead we could create a static class variable (`AccountManager.portForwardClosed`) to track it.

Reply:
done